Client/Frontend/Browser/TabManager.swift

```diff
@@ -232,9 +232,10 @@ class TabManager : NSObject {
     }
 
     @available(iOS 9, *)
-    private func addTab(request: NSURLRequest? = nil, var configuration: WKWebViewConfiguration! = nil, flushToDisk: Bool, zombie: Bool, isPrivate: Bool) -> Browser {
+    private func addTab(request: NSURLRequest? = nil, configuration: WKWebViewConfiguration! = nil, flushToDisk: Bool, zombie: Bool, isPrivate: Bool) -> Browser {
         assert(NSThread.isMainThread())
 
+        var configuration: WKWebViewConfiguration! = configuration
```

Comment:
Nit: This seems weird that we are force unwrapping a param that defaults to nil. Should we change this to be an optional + guard?